### PR TITLE
Adds mu4e-alert and mu4e-conversation.

### PR DESCRIPTION
**Pull request by innerout**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -759,6 +759,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
      - [[https://www.emacswiki.org/emacs/CategoryGnus][Gnus]] - =[built-in]= Reading e-mail and Usenet news.
      - [[https://www.emacswiki.org/emacs/MessageMode][Messages]] - =[built-in]= Composing and sending e-mail inside Emacs.
      - [[http://www.djcbsoftware.nl/code/mu/mu4e.html][mu4e]] - An e-mail client for Emacs.
+       - [[https://github.com/iqbalansari/mu4e-alert/tree/91f0657c5b245a9de57aa38391221fb5d141d9bd][mu4e-alert ]]- Desktop Notification for mu4e.
```
Comment (agsdot) [inline README.org]: @innerout , modify the URL to: https://github.com/iqbalansari/mu4e-alert . 

Also, ideally, the description after the URL matches or slightly modifies the description provided on the repo: "Desktop notifications and modeline display for mu4e"

File: README.org [hunk 2]
```diff
@@ -759,6 +759,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
...
+       - [[https://gitlab.com/ambrevar/mu4e-conversation/tree/98110bb9c300fc9866dee8e0023355f9f79c9b96][mu4e-conversation ]]- Alternative view for displaying emails in mu4e.
```
Comment (agsdot) [inline README.org]: @innerout , modify the URL to: https://gitlab.com/ambrevar/mu4e-conversation

**Comment by agsdot** [comment]
Sidenote: for the git commit "Update my fork from official repo", why is that git commit there? 

Please rebase on upstream/awesome-emacs to make a less "noisy" pull request. Thank you!

**Comment by innerout** [comment]
Thanks for the feedback i could not figure out why CI failed.

**Comment by agsdot** [comment]
Thanks for your contributions once again @innerout . On your next PR, can you first "rebase" on upstream/awesome-emacs (github.com/emacs-tw/awesome-emacs) and then put your new commits in. It looks like https://github.com/innerout/awesome-emacs/commit/bc306a26d9a2f86ec91b66a24df0c19ef3183d82 was a "merge" commit. 

I like putting in a cleaner git history for the commits.

**Comment by innerout** [comment]
Sorry i am not really familiar with the concept of rebase. Do you have a guide or an article to explain the concept properly?

**Comment by agsdot** [comment]
Great question @innerout , I don't have a quick answer or tutorial to reference right now (a websearch / google search of rebasing will bring up some nice articles and or Stack Overflow answer). I will keep your question in mind. 

I'll see if I can circle back to your question here in https://github.com/emacs-tw/awesome-emacs/pull/354 later.

**Pull request merged**